File: Bella/Bella.Services/Database/FacialHair.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Bella.Services.Database
{
    public class FacialHair
    {
        [Key]
        public int Id { get; set; }
        
        [Required]
        [MaxLength(200)]
        public string Name { get; set; } = string.Empty;
        
        public byte[]? Image { get; set; }
        
        [Required]
        public decimal Price { get; set; }
        
        public bool IsActive { get; set; } = true;

        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    }
}

File: Bella/UI/bella_hairdresser_mobile/lib/model/facial_hair.dart
import 'package:json_annotation/json_annotation.dart';

part 'facial_hair.g.dart';

@JsonSerializable()
class FacialHair {
  final int id;
  final String name;
  final String? image;
  final double price;
  final bool isActive;
  final DateTime createdAt;

  FacialHair({
    this.id = 0,
    this.name = '',
    this.image,
    this.price = 0.0,
    this.isActive = true,
    required this.createdAt,
  });

  factory FacialHair.fromJson(Map<String, dynamic> json) =>
      _$FacialHairFromJson(json);
  Map<String, dynamic> toJson() => _$FacialHairToJson(this);
}

